### PR TITLE
Add UsernsMode to container config

### PR DESCRIPTION
**Pull request by liron-l**
Signed-off-by: Liron Levin <liron@twistlock.com>

### CODE REVIEW

**Comment by cpuguy83** [comment]
Can you link to a PR/Issue where this is relevant?

File: types/container/host_config.go [hunk 1]
```diff
@@ -256,6 +256,7 @@ type HostConfig struct {
 	StorageOpt      []string          // Storage driver options per container.
 	Tmpfs           map[string]string `json:",omitempty"` // List of tmpfs (mounts) used for the container
 	UTSMode         UTSMode           // UTS namespace to use for the container
+	UsernsMode      UsernsMode        // The user namespace to user for the container
```
Comment (cpuguy83) [inline types/container/host_config.go]: 2nd `user` should be `use`

**Comment by liron-l** [comment]
Thanks @cpuguy83 fix the comment, the issue is docker/docker#20111.

**Comment by calavera** [comment]
LGTM

**Comment by runcom** [comment]
LGTM

**Pull request merged**